docs/phases/03-phase-home-metrics.md

```
@@ -0,0 +1,406 @@
+
+```
+┌─────────────────────────────────────┐
+│ 📅 今週のトライ: セサミオイルで...  │
+└─────────────────────────────────────┘
+```
+
+- 1行にコンパクトに表示
+- タップで展開 or 詳細画面へ
+
+### 曜日判定
+
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)